release_process.md

```diff
@@ -0,0 +1,52 @@
+# Sinopia Release Process
+
+This is meant to be a guide to DLSS Infrastructure Team on how to release Sinopia to stage and production. This includes the Sinopia Editor and related components (e.g., Sinopia API).
+
+These instructions apply for major releases and dependency updates.
+
+## Checklist
+1. _Skip for a dependency update:_ Product owner prepares release notes.
+  - [ ] Updates `NewsItem.js` component.
+  - [ ] Sinopia [Wiki](https://github.com/LD4P/sinopia/wiki/Latest-Release,-What's-Next)
+2. Create Sinopia Editor release.
+  - [ ] Pull latest main.
+  - [ ] Check out a branch.
+  - [ ] Update the version in *package.json*
+  - [ ] `npm i` to regenerate *package-lock.json* (see note below)
+  - [ ] `npm publish` to publish the version to [npm registry](https://npmjs.com).
+  - [ ] Commit change to branch and push and create PR for version bump.  (You can't push directly to main, since it's a protected branch).
+  - [ ] _Once PR is merged:_ Publish a [new release](https://github.com/LD4P/sinopia_editor/releases/new) with a version like `v1.0.2`.
+  - [ ] Wait for Circleci to complete building and pushing [docker images](https://hub.docker.com/repository/registry-1.docker.io/ld4p/sinopia_editor/tags?page=1&ordering=last_updated).
+3. _If changes since last release:_ Create [Sinopia Indexing Pipeline](https://github.com/LD4P/sinopia_indexing_pipeline) release.
+  - [ ] Publish a [new release](https://github.com/LD4P/sinopia_indexing_pipeline/releases/new) with a version like `v1.0.2`.
+  - [ ] Wait for Circleci to complete building and pushing [docker images](https://hub.docker.com/repository/registry-1.docker.io/ld4p/sinopia_indexing_pipeline/tags?page=1&ordering=last_updated).  
+4. _If changes since last release:_ Create [Sinopia Exporter](https://github.com/LD4P/sinopia_exporter) release.
+  - [ ] Publish a [new release](https://github.com/LD4P/sinopia_exporter/releases/new) with a version like `v1.0.2`.
+  - [ ] Wait for Circleci to complete building and pushing [docker images](https://hub.docker.com/repository/registry-1.docker.io/ld4p/sinopia_exporter/tags?page=1&ordering=last_updated).    
```

Comment:
Given the DockerHub UI is kinda stupid, we might want to add a note here about needing to refresh the page to *actually* see the last updated tags at the top.

Reply:
This comment applies to all the codebases, I should be clear.

Reply:
Added a note.

Reply:
Thank you!